MineInAbyss/src/main/java/com/derongan/minecraft/mineinabyss/Relic/RelicUseListener.java

```diff
@@ -138,7 +139,9 @@ public void onPlayerUseItem(PlayerInteractEvent playerInteractEvent) {
                 ((UseRelicBehaviour) type.getBehaviour()).onUse(playerInteractEvent);
             } else {
                 // Cancel events the relic shouldn't handle
-                playerInteractEvent.setCancelled(true);
+				if (!(type.getBehaviour() instanceof ConsumeRelicBehaviour)) { //let eat events continue
```

Comment:
Consume requires a use first? Interesting. 

Maybe we should look at handling PlayerEvent instead of individual calls per different type hrm.